.travis.yml

```diff
@@ -6,25 +6,37 @@ language: node_js
 
 matrix:
   fast_finish: true
+
+before_install: scripts/travis-before-install.sh
+
+before_script: scripts/travis-before-script.sh
+
+jobs:
   include:
-    - node_js: '9'
+    - stage: lint
+      node_js: '8'
+      env: TARGET=lint
+      script: npm start $TARGET
```

Comment:
I am guessing this can be simplified into something like

```yml
script: npm start $TARGET
node_js: '9'
jobs:
   include:
     - stage: lint
        env: TARGET=lint
     - stage: test # use default node version
        env: TARGET=test.node COVERAGE=true
     - stage: test
        node_js: '8'
        env: TARGET=test.node 
     - stage: test
        node_js : '6'
        env: TARGET=test.node
```

see https://docs.travis-ci.com/user/build-stages/#Build-Stages-and-Build-Matrix-Expansion